math/ceres-solver/Portfile

```diff
@@ -18,13 +18,11 @@ long_description    Ceres Solver is an open source C++ library for modeling \
 homepage            http://ceres-solver.org
 master_sites        ${homepage}
 
-checksums           rmd160  ce24b67679ff25c0adfe3a9cb9a72898ab57d4d5 \
-                    sha256  f7d74eecde0aed75bfc51ec48c91d01fe16a6bf16bce1987a7073286701e2fc6 \
-                    size    3802187
+checksums           rmd160  8cf4b7e69089429a0e78d3f594bda85cb398f585 \
+                    sha256  48b2302a7986ece172898477c3bcd6deb8fb5cf19b3327bc49969aad4cede82d \
+                    size    7635532
 
-# Version 2.1.x requires a Cxx 2014 compiler
-# Future version 2.2.x, will require a Cxx 2017 compiler
-compiler.cxx_standard 2014
+compiler.cxx_standard 2017
 
 # Avoid Xcode Clang 7/8: error: no viable conversion between unique_ptr types
 compiler.blacklist-append \
```

Comment:
I haven't checked but is this manually blacklisting still needed after setting the CXX standard to 2017?

Reply:
I'm not sure how compiler selection works in MacPorts and whether Clang 8 would ever be selected for C++17. I'd need your help on this.

But Clang 8 _does_ support C++17 according to https://clang.llvm.org/cxx_status.html

Let me know how to proceed.

Reply:
@reneeotten From a skim of that page, it seems to me that the blacklist is still necessary. Please let me know if you still require any changes before merging this.